### ISSUE TITLE
Generate error when attempting to delete an enum member

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#7026]
The following generates a runtime error and should be flagged as an error by a type checker.

```python
from enum import Enum

class Color(Enum):
    RED = 1
    GREEN = 2
    BLUE = 3

del Color.RED  # Type checker error
```
